account_invoice_tax_required/__init__.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 # Copyright 2015 - Camptocamp SA - Author Vincent Renaville
 # Copyright 2016 - Tecnativa - Angel Moya <odoo@tecnativa.com>
+# Copyright 2017 - Ecosoft - Kitti U. <kittiu@ecosoft.co.th>
```

Comment:
Don't add here your copyright, as you are not modifying anything. I even prefer to not put copyright in this "dummy" files.